packages/message-manager/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/message-manager",
-  "version": "7.2.0",
+  "version": "8.0.0",
```

Comment:
Since this is a major bump, shouldn't there be a "breaking" entry in the changelog?

Reply:
oh my bad this should have been minor, I updated it.